dev/tasks/tasks.yml

```diff
@@ -43,7 +43,8 @@ groups:
     - centos-*-aarch64
 
   gandiva:
-    - gandiva-*
+    - gandiva-jar-ubuntu
+    - gandiva-jar-osx
```

Comment:
Could you revert this change?
`gandiva-*` is expanded to `gandiva-ja-ubuntu` and `gandiva-jar-osx` automatically.